pulp_ansible/tests/functional/conftest.py

```diff
@@ -222,3 +222,14 @@ def _build_and_upload_collection(ansible_repo=None, **kwargs):
         return collection, collection_href[0]
 
     return _build_and_upload_collection
+
+
+@pytest.fixture
+def create_ansible_cfg(server, user):
+    return (
+        "[pulp]\nserver_list = pulp_ansible\n\n"
+        "[pulp.pulp_ansible]\n"
+        "server = {}\n"
+        "username = {}\n"
+        "password = {}"
+    ).format(server, user.username, user.password)
```

Comment:
If you want to make this a factory fixture, you need to add one more level of wrapping it in a function you can return as the fixture.